Tests/AnyLintTests/StatisticsTests.swift

```diff
@@ -7,91 +7,105 @@
     override func setUp() {
         log = Logger(outputType: .test)
         TestHelper.shared.reset()
-        Statistics.shared.reset()
+        Statistics.default.reset()
     }
 
     func testFoundViolationsInCheck() {
-        XCTAssert(Statistics.shared.executedChecks.isEmpty)
-        XCTAssert(Statistics.shared.violationsBySeverity[.info]!.isEmpty)
-        XCTAssert(Statistics.shared.violationsBySeverity[.warning]!.isEmpty)
-        XCTAssert(Statistics.shared.violationsBySeverity[.error]!.isEmpty)
-        XCTAssert(Statistics.shared.violationsPerCheck.isEmpty)
+        XCTAssert(Statistics.default.executedChecks.isEmpty)
+        XCTAssert(Statistics.default.violationsBySeverity[.info]!.isEmpty)
+        XCTAssert(Statistics.default.violationsBySeverity[.warning]!.isEmpty)
+        XCTAssert(Statistics.default.violationsBySeverity[.error]!.isEmpty)
+        XCTAssert(Statistics.default.violationsPerCheck.isEmpty)
 
         let checkInfo1 = CheckInfo(id: "id1", hint: "hint1", severity: .info)
-        Statistics.shared.found(
-            violations: [Violation(checkInfo: checkInfo1)],
-            in: checkInfo1
-        )
+        Statistics.default.found(violations: [checkInfo1: [Violation(checkInfo: checkInfo1)]])
 
-        XCTAssertEqual(Statistics.shared.executedChecks, [checkInfo1])
-        XCTAssertEqual(Statistics.shared.violationsBySeverity[.info]!.count, 1)
-        XCTAssertEqual(Statistics.shared.violationsBySeverity[.warning]!.count, 0)
-        XCTAssertEqual(Statistics.shared.violationsBySeverity[.error]!.count, 0)
-        XCTAssertEqual(Statistics.shared.violationsPerCheck.keys.count, 1)
+        XCTAssertEqual(Statistics.default.executedChecks, [checkInfo1])
+        XCTAssertEqual(Statistics.default.violationsBySeverity[.info]!.count, 1)
+        XCTAssertEqual(Statistics.default.violationsBySeverity[.warning]!.count, 0)
+        XCTAssertEqual(Statistics.default.violationsBySeverity[.error]!.count, 0)
+        XCTAssertEqual(Statistics.default.violationsPerCheck.keys.count, 1)
 
         let checkInfo2 = CheckInfo(id: "id2", hint: "hint2", severity: .warning)
-        Statistics.shared.found(
-            violations: [Violation(checkInfo: checkInfo2), Violation(checkInfo: checkInfo2)],
-            in: CheckInfo(id: "id2", hint: "hint2", severity: .warning)
+        Statistics.default.found(
+            violations: [
+                CheckInfo(id: "id2", hint: "hint2", severity: .warning):
+                    [Violation(checkInfo: checkInfo2), Violation(checkInfo: checkInfo2)]
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Multi-line collection literals should have trailing commas.](https://app.codacy.com/gh/Flinesoft/AnyLint/pullRequest?prid=5691372)